checkQC/default_config/config.yaml

```diff
@@ -12,11 +10,19 @@ parser_configurations:
     samplesheet_name: SampleSheet.csv
 
 default_handlers:
-    - name: UndeterminedPercentageHandler
-      warning: unknown
-      error: 9 # <% Phix on lane> + < value as %>
-    - name: UnidentifiedIndexHandler
-      significance_threshold: 1 # % of reads in unidentified
+  - name: UndeterminedPercentageHandler
+    warning: unknown
+    error: 9 # <% Phix on lane> + < value as %>
+  - name: UnidentifiedIndexHandler
+    significance_threshold: 1 # % of reads in unidentified
+    # Indexes which are will not cause an error even if they occur
```

Comment:
Sorry for noticing this now, but I guess a word is missing? "Indexes which are whitelisted"?

Reply:
np. Will fix that.